src/app/portlets/dot-edit-page/shared/models/layout-sidebar.model.ts

```diff
@@ -0,0 +1,3 @@
+export interface LayoutSideBar {
```

Comment:
?

Reply:
for the moment is just a placeholder

Reply:
really we did not need this interface yet (we gonna need it in the current sprint)